package.json

```diff
@@ -55,56 +55,62 @@
   },
   "devDependencies": {
     "@monastic.panic/component-playground": "*",
-    "babel": "^5.4.3",
-    "babel-core": "^5.8.34",
-    "babel-eslint": "^4.1.3",
-    "babel-loader": "^5.0.0",
-    "babel-plugin-object-assign": "1.1.0",
+    "babel-cli": "^6.10.1",
+    "babel-core": "^6.9.1",
+    "babel-eslint": "^6.0.4",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-object-assign": "1.2.1",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
     "chai": "^3.5.0",
     "chai-as-promised": "^5.3.0",
-    "component-metadata-loader": "^2.0.0",
+    "cldr-data": "^29.0.1",
+    "component-metadata-loader": "^2.0.3",
     "component-playground": "jquense/component-playground",
-    "css-loader": "^0.9.1",
-    "eslint": "^0.24.1",
-    "eslint-plugin-react": "^3.5.1",
-    "file-loader": "^0.8.1",
-    "globalize": "^0.1.1",
-    "handlebars": "^3.0.3",
-    "history": "^1.17.0",
-    "imports-loader": "^0.6.3",
-    "json-loader": "^0.5.1",
-    "karma": "~0.13.0",
-    "karma-jsdom-launcher": "^1.0.0",
-    "karma-mocha": "~0.1.9",
-    "karma-mocha-reporter": "^0.3.1",
-    "karma-phantomjs-launcher": "~0.1.4",
-    "karma-sinon-chai": "^1.1.0",
-    "karma-sourcemap-loader": "^0.3.2",
+    "css-loader": "^0.23.1",
+    "eslint": "^2.12.0",
+    "eslint-plugin-react": "^5.1.1",
+    "file-loader": "^0.8.5",
+    "globalize": "^1.1.1",
```

Comment:
globalize was intentionally left at x.1.0 the API is vastly different then which is why those bits fail. I'd put that back